connectors/src/connectors/microsoft/temporal/activities.ts

```diff
@@ -477,6 +478,7 @@ export async function syncFiles({
         dataSourceConfig,
         folderId: createdOrUpdatedResource.internalId,
         parents: [createdOrUpdatedResource.internalId, ...parents],
+        parentId: parents[0],
```

Comment:
super nit: parents[1] everywhere in connectors, except here parents[0],  is a bit confounding

Reply:
especially with parents[1] in the same file a bit below

Reply:
Indeed, I initially wrote it that way to keep the diff small, I'll change it now that it's been reviewed